docs/metrics.rst

```diff
@@ -18,7 +18,7 @@
 Metrics
 =======
 
-Airflow can be set up to send metrics to `StatsD <https://github.com/etsy/statsd>`__:
+Airflow can be set up to send metrics to `StatsD <https://github.com/etsy/statsd>`__. You must install statsd requirement: ``pip install apache-airflow[statsd]``.
```

Comment:
Is not it better to write in the next section?

Reply:
You mean in Configuration? Or adding a new "Installation" section?

Reply:
I thought about expanding the configuration section.  You can also change the name to "Setup". It will cover installation and configuration in one place.

Reply:
Makes sense! changed it.